shared/haml/hamburger.haml

```diff
@@ -119,19 +119,19 @@
     .user-links
       - if user_type == "teacher"
         - teacher_entries.each do |entry|
-          %li{class: show_teacher_options}
+          %li#teacher-entry{class: show_teacher_options}
```

Comment:
IDs should be unique, but given that this is applied to every teacher entry it won't be the case.  My upcoming change makes it easy to have unique IDs per each entry if desired, but in the meantime something similar should be done here.